test/core/test_sequencer.py

```diff
@@ -1,19 +1,35 @@
-import time
-import unittest
-from unittest.mock import MagicMock, call
-from pubsub import pub
+import pytest
 import fixate
-from fixate.core.common import TestList as FixateTL, TestClass as FixateTC
+from fixate.core.common import TestList, TestClass
+from fixate.core.checks import chk_fails, chk_passes
+from fixate.core.exceptions import SequenceAbort
+from pubsub import pub
+from unittest.mock import MagicMock, call, patch
+
 
+class MockTest(TestClass):
+    """
+    Test class that allows tracing of function calls
+    """
 
-def sleep_100m():
-    time.sleep(0.1)
-    return True
+    def __init__(self, num, mock_obj):
+        super().__init__()
+        self.mock = mock_obj
+        self.num = num
+
+    def set_up(self):
+        self.mock.test_setup(self.num)
+
+    def tear_down(self):
+        self.mock.test_tear_down(self.num)
+
+    def test(self):
+        self.mock.test_test(self.num)
 
 
-class Lst(FixateTL):
+class MockTestList(TestList):
     """
-    Mock Test List
+    Test class that allows tracing of function calls
```

Comment:
Technically a class, but can you please disambiguate from a Test Class since this is a TestList? Is this just a copy paste from before and you meant to say "Test list that allows ..."?

Reply:
Comment updated now.